dev-util/electron/electron-1.6.15.ebuild

```diff
@@ -35,6 +38,7 @@ LIBCHROMIUMCONTENT_COMMIT="a9b88fab38a8162bb485cc5854973f71ea0bc7a6"
 ASAR_VERSION="0.13.0"
 BROWSERIFY_VERSION="14.0.0"
 
+PATCHES_P="gentoo-electron-patches-${PATCHES_COMMIT}"
```

Comment:
Why not use tags? If you used `${PV}`, you wouldn't have to keep updating it.

Reply:
OK, I'll use tags.